docs/build/isc/v1.1/docs/how-tos/core-contracts/token/erc20-native-token.md

```diff
@@ -10,7 +10,7 @@ tags:
 
 # Custom ERC20 Functions
 
-Once you [registered your native token as ERC20](./erc20-native-token.md) you can use it like any other ERC20 token, with functions like
+Once you [registered your native token as ERC20](./create-native-token.md) you can use it like any other ERC20 token, with functions like
```

Comment:
Interesting find. This should have linked to ./register-token.md`. But now that create token also registers it we might want to formulate it a little different. Something like this maybe:

```suggestion
Once you [created your token](./create-native-token.md) it is automatically registered as ERC20 token and you can use it like any other ERC20 token, with functions like
```

Reply:
Updated, thanks.